features/account/customer_account/redirect_when_already_signed_in.feature

```diff
@@ -1,13 +1,20 @@
-@customer_account
 Feature: Redirect when already signed in
     In order to be aware that I am already logged in
     As a Customer
-    I want to be redirected to account panel dashboard when accessing the login page
+    I want to be redirected to account panel dashboard when accessing the login page or the registration page
 
-    @ui @no-api
+    @customer_account @ui @no-api
     Scenario: Trying to access login page as logged in user
         Given the store operates on a single channel in "United States"
         And there is a customer "Francis Underwood" identified by an email "francis@underwood.com" and a password "whitehouse"
         And I am logged in as "francis@underwood.com"
         When I want to log in
         Then I should be redirected to my account dashboard
+
+    @customer_registration @ui @no-api
```

Comment:
I support restoring the original common tag. I think this registration scenario fits the `@customer_account` tag since the customer is logged in.